apis/r/R/TileDBObject.R

```diff
@@ -38,6 +38,13 @@ TileDBObject <- R6::R6Class(
       private$.tiledbsoma_ctx <- tiledbsoma_ctx
       private$.tiledb_ctx <- self$tiledbsoma_ctx$context()
 
+      # TODO: re-enable once new UX is worked out
+      # soma_context <- soma_context %||% soma_context()
```

Comment:
I think we want this active to be able to test as we just did with different regions.  So let's expand the signature, and label the accessor as temporary.  It's not in the 'official' user facing factory method.

Reply:
We may also want to add the 'existence proof test' in a test file, possibly commented out (and we don't call aws, usually) -- but for illustration?

Reply:
Because `soma_context()` caches globally, my thought process was that we could use the global context for modifications without changing the signature

```r
soma_context(config = c(vfs.s3.region = "us-west-2"))
grp <- TileDBGroup$new(uri, ...)
```
If you feel strongly otherwise, I can change this

Reply:
Ah, fair, but that is kinda a hidden feature I will have to take out at some point?

But agreed, for now it can stay.